aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/UploadTask.java

```diff
@@ -88,10 +88,15 @@ public UploadTask(TransferRecord uploadInfo,
      */
     @Override
     public Boolean call() throws Exception {
-        if (!isNetworkHandlerRegisteredAndNetworkConnected()) {
```

Comment:
The `isNetworkHandlerRegisteredAndNetworkConnected` method is also used in  the `uploadMultipartAndWaitForCompletion` method. It would be good make this change there as well.

Also, once that is done, remove the `isNetworkHandlerRegisteredAndNetworkConnected` method as it will no longer be needed.

Reply:
Made the change.